jaas-login/src/main/java/com/wipro/ats/bdre/jaas/login/JAASUserPrincipal.java

```diff
@@ -59,15 +59,12 @@ public int hashCode() {
     public boolean equals(Object obj) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "equals" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)